beam/www/beam/components/Transfer.vue

```diff
@@ -24,7 +27,7 @@ const store = useDataStore()
 const listItems = ref<ListViewItem[]>([])
 const stockEntryId = ref('')
 
-const create = async () => {
+const create = async (): Promise<{ data: any }> => {
```

Comment:
This should already be typed to return a `StockEntry` object so I don't think you need an explicit response:

![image](https://github.com/user-attachments/assets/d9643dd3-1349-4b75-93a6-bc9b57df2fc4)